src/main/resources/default.properties

```diff
@@ -1,2 +1,4 @@
-spring.data.mongodb.database=Projects
-server.port=8081
\ No newline at end of file
+spring.data.mongodb.database=intern_project
+storage.root=/intern-project
+storage.file.max-size.mb=500
+server.port=0
```

Comment:
random port is applicable only in test environment (`src/test/resources/default.properties`). Here it is better 8080 default port, so you can just remove this line